world-server/src/world/player/Player.ts

```diff
@@ -12,6 +12,7 @@ export class Player extends Entity {
     public weapon: Weapon;
     public score: number = 0;
     public direction: number = 0;
+    public initialposition: Vector2;
```

Comment:
initialPosition...
How is it related to chat? Why do we actually need to store it?

Reply:
The guys working on that seemed to forgot to checkout to a separate branch when working on the respawn functionality. Removed.